godtools/App/Services/Renderer/Views/Tool/Views/Page/ToolPageViewModel.swift

```diff
@@ -25,7 +25,7 @@ class ToolPageViewModel: MobileContentPageViewModel {
         self.analytics = analytics
         self.hidesCallToAction = pageModel.isLastPage
                 
-        self.analyticsEventsObjects = MobileContentAnalyticsEvent.initAnalyticsEvents(
+        self.visibleAnalyticsEventsObjects = MobileContentAnalyticsEvent.initAnalyticsEvents(
```

Comment:
It seems I duplicated some of this logic here for tracking page analytics in the mobile content renderer.  I'm going to create a ticket and look into consolidating some of this.